packages/api-explorer/src/components/DocSdkUsage/DocSdkUsage.tsx

```diff
@@ -40,31 +48,59 @@ interface DocSdkUsageProps {
  *  links to the source files
  */
 export const DocSdkUsage: FC<DocSdkUsageProps> = ({ method }) => {
-  const tabs = useTabs()
   const { examples } = useContext(LodeContext)
   const languages = findExampleLanguages(examples, method.name)
   if (languages.length === 0) return <></>
 
+  const tableExamples = prepareExampleTableData(
+    languages,
+    examples,
+    method.operationId
+  )
+
   return (
     <CollapserCard id="examples" heading="SDK Examples">
-      <>
-        <TabList {...tabs}>
-          {languages.map((language) => (
-            <Tab key={language}>{language}</Tab>
-          ))}
-        </TabList>
-        <TabPanels {...tabs} pt="0">
-          {languages.map((language) => (
-            <TabPanel key={language}>
-              <DocExamples
-                lode={examples}
-                language={language}
-                operationId={method.operationId}
-              />
-            </TabPanel>
-          ))}
-        </TabPanels>
-      </>
+      <Card height="auto" p="medium" mt="small">
+        <Flex alignItems="center" mx="medium" minHeight="2rem">
+          <FlexItem flexBasis="50%">
+            <Text fontWeight="semiBold">Filename</Text>
+          </FlexItem>
+          <FlexItem flexBasis="40%">
+            <Text fontWeight="semiBold">Language</Text>
+          </FlexItem>
+          <FlexItem flexBasis="10%">
+            <Text fontWeight="semiBold" style={{ float: 'right' }}>
+              Line
+            </Text>
+          </FlexItem>
+        </Flex>
+        {tableExamples.map((exampleRow: ExampleLinkTableRow, i: number) => (
+          <div key={i}>
```

Comment:
I believe the use of the `div` here is due to the need for a keyed fragment. If that's the case, I suggest using 
`<React.Fragment key={key}></React.Fragment>`. @bryans99 in case he has other opinions or flat out disagrees with me 😄 

I also get a bit nervous about using the index as a key after being bitten by it a few times. Perhaps it's not a big deal on non input components but you could consider using something more unique, such as `${exampleRow.filename}#${exampleRow.line}`.

Reply:
Use React.Fragment rather than div. Use unique data from example row permalink, filename + line + column? Is that unique enough.